### PR TITLE
Improve list of commands for Optelec displays in the user guide

### LINKED ISSUE #8037: Uninstalling avast antivirus with NVDA!
When trying to uninstall avast antivirus, with NVDA using the avast uninstall utility, it asks you if you want to uninstall avast with out going into safe mode, whitch is not recommended, and when I chews the button that says no, I get a message saying, you must disable the self protection module in avast antivirus, under settings in the troubleshooting tab, now when I do it with jaws, I chose the button that says no, and avast uninstalls, and asks me to reboot my machine!

### LINKED ISSUE #8073: UG at section 14.2 (Optelec BC6...) there are 2 separate entries for alt key and one space is missing in row where command for enter key was written
### Steps to reproduce:

- open an userguide of nvda
- go to section 14.2
- look at the provided table
- see rows 16 and 34
- for enter see row 23

### Expected behavior:

there should be only one row for alt key with sp2 and alt written in the right column and "|" should not be shown in the table when a UG was opened in browser

### Actual behavior:

there are two rows for alt key at section for Optelec bc6 instead of one in nvda's userguide and "|" is shown after listing commands for emulation of enter key

### System configuration:
NVDA version:

next: 14941,891a9c9e
NVDA Installed or portable:
installed

### PR DESCRIPTION
**Pull request by lukaszgo1**
### Link to issue number:
Fixes #8073
### Summary of the issue:
In the table containing  keyboard commands for alva displays in the section 14.3 of the user guide there was two entries for the alt key, one space was missing and when the command has more than one key assigned comma and "or" was used to separate them.

  ### Description of how this pull request fixes the issue:
1. Two entries for an alt key was consolidated into one.
2. additional space added after enter key.
3. replaced "or" with commas to make table consistent with the rest of the user guide.

### Testing performed:
Compiled user guide and opened it in the browser.
### Known issues with pull request:
None
### Change log entry:
None needed. As far as i know changes to documentation are not listed in a changes file.

### CODE REVIEW

**Review by josephsl — DISMISSED** [review]
Excellent, thanks for catching spacing problem on one line.

**Comment by josephsl** [comment]
Hi,

@MichaelDCurran, requesting permission to merge this into master, or if not, permission to incubate. Thanks.

**Comment by leonardder** [comment]
Note that this is not interfering with #8230 as far as I can see.


[**@lukaszgo1**](https://github.com/lukaszgo1) commented on [29 Jun 2018, 21:43 CEST](https://github.com/nvaccess/nvda/pull/8465#issue-198407453 "2018-06-29T19:43:33Z - Replied by Github Reply Comments"):
> ### Link to issue number:
> Fixes #8037

This issue seems to be unrelated

**Comment by gregjozk** [comment]
Hello,

a quoted issue is #8073 and yes, this is only a UG issue.
thanks.

regards,

**Comment by josephsl** [comment]
Hi,

Requesting permission for this to go to master. Thanks.

**Comment by lukaszgo1** [comment]
@gregjozk wrote
>a quoted issue is #8073 and yes, this is only a UG issue.

Thanks for pointing this out. I've fixed a PR description.

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -2006,15 +2006,15 @@ Please see the display's documentation for descriptions of where these keys can
...
 | escape key | sp3 |
 | tab key | sp4 |
 | upArrow key | spUp |
 | downArrow key | spDown |
 | leftArrow key | spLeft |
 | rightArrow key | spRight |
-| enter key | spEnter, enter|
-| Report date/time | sp2+sp3 |
+| enter key | spEnter, enter |
+| Report date/time | sp1+sp2 |
```
Comment (leonardder) [inline user_docs/en/userGuide.t2t]: Could you please revert this change?

**Review by LeonarddeR — CHANGES_REQUESTED** [review]

File: user_docs/en/userGuide.t2t (hunk 1 above)
Comment (lukaszgo1) [reply user_docs/en/userGuide.t2t]: @leonardder Done. I'm a real newbie to git, so i might have done somethiing wrong, but it looks like it is working.

**Review by LeonarddeR — APPROVED** [review]
@michaeldcurran: I've changed the base branch of this pr to beta, since IMO, it makes sense to have these issues resolved as part of 2018.3.

**Review by michaelDCurran — APPROVED** [review]
